.github/actions/upload-nightly/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "Upload Nightly"
+description: "Upload file to the nightly repository"
+
+inputs:
+  repository_url:
+    description: "The URL of the repository"
+    required: true
+  repository_username:
+      description: "Repository username"
+      required: true
+  repository_password:
```

Comment:
We don't need to create inputs for passwords, it is passed in the ENV.

Reply:
are they, where?